R/read_txt_Renishaw.R

```diff
@@ -164,48 +164,47 @@ read_txt_Renishaw <- function(file = stop("file is required"),
 
 hySpc.testthat::test(read_txt_Renishaw) <- function() {
   context("read_txt_Renishaw")
+  path <- system.file("extdata/fileio/txt.Renishaw", package="hySpc.read.txt")
+  paracetamol <- paste0(path, "/paracetamol.txt")
+  laser <- paste0(path, "/laser.txt.gz")
+  chondro <- paste0(path, "/chondro.txt")
```

Comment:
Can you, please, not use the names `paracetamol`, `laser`, and `chondro` for file names. Use  `f_paracetamol`, `f_laser`, and `f_chondro` instead.